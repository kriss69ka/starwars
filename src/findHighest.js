export const findHighest = (data) => Math.max(...data.map(item => parseInt(item.height, 10)))