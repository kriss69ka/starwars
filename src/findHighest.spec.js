import {findHighest} from "./findHighest";
import data from "./Data.json";

it('returns highest person', () => {
    expect(findHighest(data)).toMatchSnapshot();
})