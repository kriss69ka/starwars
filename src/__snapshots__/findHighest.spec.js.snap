// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns highest person 1`] = `202`;
